src/app/coursegrab/models/semester.py

```diff
@@ -0,0 +1,14 @@
+from app import db
+
+
+class Semester(db.Model):
+    __tablename__ = "semesters"
+    current_semester = db.Column(db.String, primary_key=True)
+
+    def __init__(self, **kwargs):
+        self.current_semester = kwargs.get("current_semester")
+
+    def serialize(self):
```

Comment:
is the serialize method necessary? once we query the semester we can just do `semester.current_semester` to get the single field